pkg/cleanup/cleanup.go

```diff
@@ -62,6 +65,40 @@ func GetInactiveTags(activeTags, tags *[]string) []string {
 	return inactiveTags
 }
 
+// GetOrphanTags returns the tags that does not have any git commit match
+func GetOrphanTags(values, tags *[]string, matchOption MatchOption) []string {
+	var orphans []string
+
+	log.WithField("values", values).Debug("values")
+	log.WithField("tags", tags).Debug("tags")
```

Comment:
What tags and values? Git tags, image tags, ...? Even Debug log messages should not be neglected in the quality of the log messages. Please provide better messages, rename the "tags" variable to "gitTags" or "imageTags" (depending what it is) and be consistent in that naming.